src/pages/index.js

```diff
@@ -84,12 +188,31 @@ const popupWithFormCard = new PopupWithForm(
   "#card-add-modal",
   handleCardAddSubmit
 );
+
 popupWithFormCard.setEventListeners();
 
 cardAddButton.addEventListener("click", () => {
   popupWithFormCard.open();
 });
 
+//Avatar Change
+const handleAvatarEditSubmit = (input) => {
+  api.setUserAvatar(input).then(() => {
+    avatarEditButton.src = input.avatar;
```

Comment:
Needs correction: it's needed to use UserInfo class methods for updating the avatar